src/nk_vision_pkg/launch/nk_vision.launch.py

```diff
@@ -0,0 +1,148 @@
+        #                  'publish_rate': 50.0,
+        #                  'camera_info_url': 'package://nk_vision/config/arducam.yaml',
+        #                  'frame_id': 'camera_2'}],
+        #     remappings=[('image_raw', '/camera_2/image_raw'),
+        #                 ('camera_info', '/camera_2/camera_info')]),g
+
+        # launch_ros.actions.Node(
+        #     package='image_publisher', executable='image_publisher_node', output='screen',
+        #     arguments=[device_1],
+        #     parameters=[{'use_sim_time': use_sim_time,
+        #                  'publish_rate': 50.0,
+        #                  'filename': device_1,
+        #                  'camera_info_url': 'package://nk_vision/config/arducam.yaml',
+        #                  'frame_id': 'camera_1'}],
+        #     remappings=[('image_raw', '/camera_1/image_raw'),
+        #                 ('camera_info', '/camera_1/camera_info')]),
+
+        # launch_ros.actions.Node(
+        #     package='rviz2', executable='rviz2', output='screen',
+        #     arguments=['-d', get_package_share_directory('nk_vision') + '/config/config_file.rviz']
+        # ),
+
+        # IncludeLaunchDescription(AnyLaunchDescriptionSource(
+        #         get_package_share_directory('nk_vision') + '/launch/aruco_tracker_1.launch.xml')),
+
+        # IncludeLaunchDescription(AnyLaunchDescriptionSource(
+        #         get_package_share_directory('nk_vision') + '/launch/aruco_tracker_2.launch.xml')),
+
+        # IncludeLaunchDescription(AnyLaunchDescriptionSource(
+        #         get_package_share_directory('frc_2024_field_description') + '/launch/main.launch.py')),
+        
+        # IncludeLaunchDescription(AnyLaunchDescriptionSource(
+        #         get_package_share_directory('robot_2024_description') + '/launch/main.launch.py')),
+
+        # # launch_ros.actions.Node(
+        # #     package='nk_vision', executable='pose_estimation.py', output='screen'),
+        
+        # launch_ros.actions.Node(
+        #     package='nk_vision', executable='tf2network_table.py', output='screen',
+        #     parameters=[{'transfer_topics': ["base_link_1", "base_link_2", "base_link_3"]}]),
+        
+        # launch_ros.actions.Node(
+        #     package='nk_vision', executable='network_table2tf.py', output='screen',
+        #     parameters=[{'transfer_topics': ["base_link"]}]),
+
+        # ExecuteProcess(
+        #     cmd=['ros2', 'bag', 'record', 
+        #          '/tf',
+        #          '/camera_1/robot_description',
+        #          '/camera_2/robot_description',
+        #          '/robot_description',
+        #          '/joint_states',
+        #          '/tf',
+        #           '/field/robot_description',
+        #           '-o',
+        #           '/home/nasa-knights/vision_logs/data'],
+        #     output='screen'
+        # ),
+
+        IncludeLaunchDescription(PythonLaunchDescriptionSource(
```

Comment:
Remove this block and instead include the component launch from the ICP object detect package.